xml/admin_ceph_troubleshooting.xml

```diff
@@ -515,4 +515,101 @@ keep_jobs: 1</screen>
    </procedure>
   </tip>
  </sect1>
+ <sect1 xml:id ="ceph-fix-too-many-pgs">
+   <title>Too Many PGs Per OSD</title>
+   <para>
+     The <option>TOO_MANY_PGS</option> flag is raised when the number of PGs in
+     use is above the configurable threshold of <option>mon_pg_warn_max_per_osd</option>
+     PGs per OSD. If this threshold is exceeded, the cluster does not allow new pools
+     to be created, pool <option>pg_num</option> to be increased, or pool
+     replication to be increased.
+   </para>
+   <para>
+     &productname; &prevproductnumber; and &productnumber; have two ways to
+     solve this issue.
+   </para>
+   <procedure>
+     <title>Solution 1</title>
+     <step>
+       <para>
+         Set the following in your <filename>ceph.conf</filename>:
+       </para>
+<screen>
+[global]
+
+mon_max_pg_per_osd = 800  #  depends on your amount of PGs
+osd max pg per osd hard ratio = 10 #  the default is 2. We recommend to set at least 5.
+mon allow pool delete = true # without it you can't remove a pool
+</screen>
+     </step>
+     <step>
+       <para>
+         Restart all MONs and OSDs one by one.
+       </para>
+     </step>
+     <step>
+       <para>
+         Check the value of your MON and OSD ID:
+       </para>
+<screen>&prompt.cephuser;ceph --admin-daemon /var/run/ceph/ceph-mon.<replaceable>ID</replaceable>.asok config get  mon_max_pg_per_osd
+&prompt.cephuser;ceph --admin-daemon /var/run/ceph/ceph-osd.<replaceable>ID</replaceable>.asok config get osd_max_pg_per_osd_hard_ratio
+</screen>
+     </step>
+     <step>
+       <para>
+         Execute the following to determine your default <option>pg_num</option>:
+       </para>
+<screen>&prompt.cephuser;rados lspools
+&prompt.cephuser;ceph osd pool get <replaceable>USER-EMAIL</replaceable> pg_num
+</screen>
+     </step>
+     <step>
+       <para>
+          <emphasis>With caution</emphasis>, execute the following commands to remove pools:
+       </para>
+<screen>&prompt.cephuser;ceph osd pool create <replaceable>USER-EMAIL</replaceable>new 8
+&prompt.cephuser;rados cppool <replaceable>USER-EMAIL</replaceable> default.rgw.lc.new
```

Comment:
How long does this take and where does the data move/copy happen? Is the rados tool reading and writing all the data so that the whole data moves the wire to the admin machine?

Reply:
Before we start the copy - how can we stop the client I/O that modifies objects in the given pool?